3001-4000/LIT3311Prosph.xml

```diff
@@ -50,12 +79,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <term key="Aks"/>
             </keywords>
          </textClass>
+         <abstract>
+            <p>A letter (<foreign xml:lang="la">prosphoneticus</foreign> ) by <persName
+                  ref="PRS3255CyrilI"/> addressed to Princesses Arcadia and Marina, sisters of
+                  <persName ref="PRS9480Theodosi"/>. The letter, originally composed in Greek
```

Comment:
Yes, I think this looks good! CPG talks specifically about the Geez version, correct?

Maybe one could then also add CPG to the bibliography below (under the category `<listBibl type="secondary">`)?

Reply:
If we have the same text in CPG, with a number, this should have a `listBibl type="clavis`" for mapping

Reply:
https://betamasaheft.eu/Guidelines/?id=work-teiHeader example 4